src/main/java/org/jose/F.java

```diff
@@ -24,4 +29,25 @@ private int fibonaccirecursivo(int n) {
         }
     }
 
-}
+    private int fibonaccibottom_up(int n) {
```

Comment:
method names should only contain letters, and use camel case. in camel case everywhere you'd put a space, dash and the like you use uppercase instead: `fibonacciBottomUp`